### PR TITLE
Feature/linked-navbar-items

### PR DESCRIPTION
**Pull request by marcos-de-barros-vioque**
Hi!
I've created the subpages and linked them in the navbar.
Thanks for reviewing!

### CODE REVIEW

File: components/Layout.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import {WelcomeCard} from "../components/WelcomeCard";
 import styled from "styled-components";
 import {Icon} from "@iconify/react";
+import Link from "next/link";
+import {WelcomeCard} from "../components/WelcomeCard";
 
 export function Layout() {
   return (
```
Comment (actyralf) [inline components/Layout.js:7]: You don't render children in the Layout component, that's why you don't see the actual page content of your different pages. Change the function signature like this: `export function Layout({children})` and place the `{children}` element at the appropriate place of your layout (I guess between `WelcomeCard` and `NavBar`)

**Review by actyralf — CHANGES_REQUESTED** [review]
Hi Marcos,

well done, but your page content does not show up because of a bug in your Layout component. Please check comment below.

File: components/Layout.js (hunk 1 above)
Comment (marcos-de-barros-vioque) [reply components/Layout.js:7]: Thanks @actyralf ! Ok, the logo and the welcome message are displayed on all pages, and they should only be displayed on the home page

**Review by MeenaFoerderer — CHANGES_REQUESTED** [review]
Hey Marcos 🙂
in your `WelcomeCard.js` component you use ` <br />` for setting margins. Better style the `Image` component accordingly, i.e. giving it a margin. I also checked the mobile view and saw that the image is too large for iPhone SE screen. You should also adjust the size of it, otherwise it might not pass quality assurance.

**Comment by marcos-de-barros-vioque** [comment]
Hi @MeenaFoerderer, thanks a bunch for your review! I've resized the welcome logo and adjusted styling with margin.

**Pull request merged**